### ISSUE TITLE
FE: Update VAMC Billing and Insurance Phone number to work with new fields

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17866]
## Background
In #17861 the Phone number field was updated to use the Phone paragraph type field (instead of the Telephone number field that it previously was). This means the FE query needs to be updated to pull the data properly from these new fields and then display on the FE.

**NOTE:** Drupal work won't be in main / prod. Is in an integration branch.

## Description
The Phone paragraph type has more fields than what we previously had so there are more conditions to account for:
- Type:
  - Phone: standard component, no additional props
  - Fax: the `not-clickable` prop in the component should be used
  - SMS: the `sms` prop in the component should be used 
  - TTY: the `tty` prop should in the component be used
- Phone number: This is the info that would go into the `contact` prop of the component
- Extension number: This is the info that would go into the `ext` prop of the component
- Label:
  - This is hidden from Drupal and we will not be changing the current label that is used on the FE

## Acceptance Criteria
- [x] If type Fax is selected, number is not clickable
- [x] If type SMS is selected, number uses the `sms` prop
- [x] If type TTY is selected, number shows "TTY"
- [x] Exts are displayed and parsed correctly
- [x] The existing hardcoded label displays
- [x] Requires accessibility review

```[tasklist]
### Testing & Launch
- [ ] The CMS PR from 17861 will need to be referenced in this PR
- [ ] Once the Content Build PR is ready, integrated testing of both PRs should happen on Tugboat
- [ ] Keep Content Build PR in Pending Integration until Drupal PR merges & deploys
- [ ] More launch details in Drupal ticket:
- [ ] https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17861
```

**Comment by davidmpickett** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/VAgov-Sitewide-Product-Team-6645154188a81a1f7a0a9464/issues/department-of-veterans-affairs/va.gov-cms/17866?planning-poker) with Zenhub @chriskim2311 @eselkin @randimays

**Comment by randimays** [comment]
Need @eselkin to confirm, but these 3 phone number tickets (#17865, #17867 and this ticket), share a common phone-formatting utility. We should only need to update it once and then test for all.

**Comment by davidmpickett** [comment]
Closing in favor of an integrated full stack story #17861

**Comment by jilladams** [comment]
This has been deployed to prod behind a flipper. We've done everything we can to complete the work, so am closing ticket. Revealing it in production will happen via #19064 launch plan.
